packages/blade/src/components/Drawer/docs/Drawer.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import type { StoryFn } from '@storybook/react';
+import { Title } from '@storybook/addon-docs';
+import type { DrawerProps } from '../';
+import { Drawer, DrawerBody, DrawerHeader } from '../';
+import { DrawerStackingStory } from './stories';
+import { Box } from '~components/Box';
+import { Button } from '~components/Button';
+import { DownloadIcon } from '~components/Icons';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import { Heading } from '~components/Typography';
+import { Badge } from '~components/Badge';
+import { TextInput } from '~components/Input/TextInput';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentName="Drawer"
+      componentDescription="A drawer is a panel that slides in mostly from right side of the screen over the existing content in the viewport."
+      figmaURL="https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=78667%3A66659&mode=design"
+    >
+      <Title>Usage</Title>
+      <Sandbox>{DrawerStackingStory}</Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/Drawer',
+  component: Drawer,
+  subcomponents: { Drawer, DrawerHeader, DrawerBody },
```

Comment:
We need to do this for all overlay components.

Reply:
The con of this is that the storybook controls don't work with such components. With Drawer it seems fine since it doesn't have that many props and I have added interactive examples but don't think so we can do it on other components